yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -201,41 +201,47 @@ func (adb *AssessmentDB) BulkInsert(table string, records [][]string) error {
 }
 
 const (
-	InsertTableStats = `INSERT INTO %s (schema_name, object_name, row_count, column_count, reads, writes, is_index, object_type, parent_table_name, size_in_bytes)
+	InsertTableStats = `INSERT INTO %s (schema_name, object_name, row_count, column_count, reads, writes, reads_per_second, writes_per_second, is_index, object_type, parent_table_name, size_in_bytes)
 	SELECT 
 		trc.schema_name,
 		trc.table_name AS object_name,
 		trc.row_count,
 		tcc.column_count,
-		tii.seq_reads AS reads,
-		tii.row_writes AS writes,
+		COALESCE(tii.seq_reads, 0) AS reads,
+		COALESCE(tii.row_writes, 0) AS writes,
+		0 AS reads_per_second,
```

Comment:
What's the purpose of these two new columns `reads_per_second` and `writes_per_second` ? What do  `reads` and `writes` mean, then?  And why are they 0?

Reply:
Oh, I see - they are later updated by `UpdateStatsWithRates` . 
I guess this was the discussion regarding 0/NULL? So are we updating this to NULL?

Reply:
right, they are later update with Rates.
Yes we are going with NULL, once the underlying sizer algo is capable of handling NULLs then i can revert my code from using `0` -> `NULL`